Homework-4-ManyToManyRelationships/src/dao/groupDao.ts

```diff
@@ -0,0 +1,84 @@
+import { PrismaClient } from ".prisma/client";
+import { Service } from "typedi";
+import type { BaseGroupDTO, GroupDTO } from "../types/groupDTO";
+
+const prisma = new PrismaClient();
```

Comment:
It's recommended to have only one prisma client:
https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/instantiate-prisma-client#the-number-of-prismaclient-instances-matters

Reply:
Will be fixed in next homework!